server/src/main/java/org/candlepin/model/AsyncJobStatus.java

```diff
@@ -126,8 +126,8 @@ public boolean isTerminal() {
 
     @ElementCollection(fetch = FetchType.EAGER)
     @CollectionTable(name = "cp_async_job_metadata", joinColumns = @JoinColumn(name = "job_id"))
-    @MapKeyColumn(name = "key")
-    @Column(name = "value")
+    @MapKeyColumn(name = "mkey")
```

Comment:
Why this change (and mval)?

Reply:
At least one appears to be a keyword in mysql, and Hibernate doesn't feel like quoting column names in the queries it generates. I'm pretty sure "value" might be fine, but I changed both just to be safe.

It probably won't end up mattering much as metadata is set to be serialized and the table removed in the other PR.

Reply:
I've reverted this and, instead, added quotes to the column names in code (see AsyncJobStatus). Not sure why Hibernate doesn't do it by default if it has a standard syntax for specifying a column name should be quoted, but whatever.